ironfish/src/account/accounts.test.slow.ts

```diff
@@ -249,6 +256,55 @@ describe('Accounts', () => {
     })
   }, 600000)
 
+  it('throws a ValidationError with an invalid expiration sequence', async () => {
+    // Initialize the database and chain
```

Comment:
nit: Lets delete this comment.. I see the old code is wrong too I'll put up a PR to clean those up as well.